pom.xml

```diff
@@ -471,6 +471,17 @@
             <version>2.6.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Can we implement these tests using existing test-utils dependencies?   It is required to check the license of the newly added dependencies and keep our dependency tree clean and each item is necessary.

Reply:
Hi, Yes I can do it with `mockito`, however `powermock` is a bit more powerful and you will be losing coverage if I used `mockito` instead